src/mlpack/methods/ann/activation_functions/elliot_function.hpp

```diff
@@ -65,24 +65,28 @@ class ElliotFunction
   /**
    * Computes the first derivative of the Elliot function.
    *
-   * @param y Input data.
+   * @param x Input activation.
+   * @param y Result of Fn(x).
    * @return f'(x).
    */
-  static double Deriv(const double y)
+  static double Deriv(const double /* x */, const double y)
   {
+    // TODO: I don't think this is correct
```

Comment:
Agreed.  I think it should be `std::pow(1.0 + std::abs(x), 2)`... or `y^2 / x^2`, but I'm pretty sure the first would be faster.  Do you want to make that change (if what I wrote is right)?